INSTALL

```diff
@@ -9,6 +9,20 @@ wiki pages:
 * If you are in a hurry *
 *************************
 
+On Linux (Debian):
+
+ Note: you will need to add the Debian 'unstable' distribution to your source
+ file first.
+
+ Add the following line to your `/etc/apt/sources.list` file:
+
+ deb http://ftp.us.debian.org/debian unstable main contrib non-free
+
+ Install the gRPC library
+ $ sudo apt-get install libgrpc-dev
```

Comment:
I think it's standard to say `[sudo]` so people need to consciously decide to run as root (e.g. copy-paste won't work as-is).